src/Control/Monad/Trans/Free.hs

```diff
@@ -422,8 +422,8 @@ instance (Monad m, Traversable m, Traversable f) => Traversable (FreeT f m) wher
 -- | Lift a monad homomorphism from @m@ to @n@ into a monad homomorphism from @'FreeT' f m@ to @'FreeT' f n@
 --
 -- @'hoistFreeT' :: ('Monad' m, 'Functor' f) => (m ~> n) -> 'FreeT' f m ~> 'FreeT' f n@
-hoistFreeT :: (Monad m, Functor f) => (forall a. m a -> n a) -> FreeT f m b -> FreeT f n b
-hoistFreeT mh = FreeT . mh . liftM (fmap (hoistFreeT mh)) . runFreeT
+hoistFreeT :: (Functor m, Functor f) => (forall a. m a -> n a) -> FreeT f m b -> FreeT f n b
+hoistFreeT mh = FreeT . mh . fmap (fmap (hoistFreeT mh)) . runFreeT
```

Comment:
`hoistFreeT` in `Control.Monad.Trans.Free.Ap` could also benefit from replacing `liftM` with `fmap`:

https://github.com/ekmett/free/blob/5721f23f711cebaa93bc6292ed025a80fbc4f06b/src/Control/Monad/Trans/Free/Ap.hs#L422-L423

Reply:
OK, I have incorporated that.